generators/entity/index.js

```diff
@@ -162,7 +162,7 @@ class EntityGenerator extends BaseBlueprintGenerator {
                 context.prodDatabaseType = configuration.get('prodDatabaseType') || this.options.db;
                 context.devDatabaseType = configuration.get('devDatabaseType') || this.options.db;
                 context.skipFakeData = configuration.get('skipFakeData');
-                context.searchEngine = configuration.get('searchEngine');
+                context.searchEngine = _.isUndefined(configuration.get('searchEngine')) ? false : configuration.get('searchEngine');
```

Comment:
I prefer this notation:
```suggestion
                context.searchEngine = _configuration.get('searchEngine')) || false;
```

Reply:
done !